src/App.test.tsx

```diff
@@ -1,12 +1,21 @@
-import { screen, render } from '@testing-library/react';
+import { screen } from '@testing-library/react';
 import App from './App';
+import render from './router/testRenderer';
 
-it('displays a +1 button', () => {
+it('App feature tour', async () => {
```

Comment:
I've observed that the test case in question involves asynchronous operations, which is quite practical, especially when dealing with components that fetch data from our API, as we are currently doing with Leandro in the Backend.

To enhance the reliability of our test suite, I suggest adding the await keyword to each expect statement we are going to update on this code. Additionally, it might be a good idea to replace the getByRole method with findByRole since it's the recommended approach for testing asynchronous elements. This adjustment will ensure that our tests wait for the expected elements to appear, contributing to more robust and reliable testing of our application's behavior.  Thanks again @Ralexs0096 , great job!

```suggestion
it('App feature tour', async () => {
 render(<App />);
  await expect(screen.findByRole('button', { name: /area/i })).toBeInTheDocument();
});
```

Reply:
In this case, it is not necessary to use the`findBy` method since the menu loads without doing any asynchronous behavior, probably it will be necessary when we integrate the calls to the API.